signature_test.go

```diff
@@ -70,17 +70,18 @@ func TestSignatureEncodings(t *testing.T) {
 		msg := CRandBytes(128)
 		sig := tc.privKey.Sign(msg)
 
-		// store as wire
-		bin, err := cdc.MarshalBinary(sig)
+		// store as amino
+		bin, err := cdc.MarshalBinaryBare(sig)
 		require.Nil(t, err, "%+v", err)
 		if tc.sigSize != 0 {
-			assert.Equal(t, tc.sigSize+4, len(bin))
+			// Q: where is 1 byte coming from?
+			assert.Equal(t, tc.sigSize+amino.PrefixBytesLen+1, len(bin))
```

Comment:
Q: where is 1 byte coming from?

Reply:
Before, go-wire used to not encode array sizes... so `type SignatureEd25519 [64]byte` used to be
just the prefix bytes + the 64 bytes, but now we encode it the same as if it were a slice which happens to have 64 bytes (e.g. is preceded by uvarint(64) which is 1 byte long).

Reply:
thanks for clarifying this